mesonpy/__init__.py

```diff
@@ -275,6 +275,9 @@ def platform_tag(self) -> str:
                 # latter specifies the target macOS version Python was built
                 # against.
                 parts[1] = platform.mac_ver()[0]
+                # Only pick up the major version
+                # https://github.com/FFY00/meson-python/issues/160
+                parts[1] = parts[1].split('.')[0]
```

Comment:
And append `'.0'` then, right? We want `11.0`, `12.0`, not `11`, `12`

Reply:
We are already doing that below.